### PR TITLE
Update to Minio GCS Gateway

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
Initial edits

- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

File: docs/gateway/gcs.md [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
-## Run Minio Gateway for GCS
-### Create service account key for GCS and get the credentials file
-1. Go to the [API Console Credentials page](https://console.developers.google.com/project/_/apis/credentials).
-2. Select your project or create a new project. Note down your project ID.
-3. On the Credentials page, select the __Create credentials__ drop-down, then select __Service account key__.
-4. From the __Service account__ drop-down, select __New service account__
-5. Fill up __Service account name__ and __Service account ID__
-6. For the __Role__, click the select dropdown to choose __Storage__ -> __Storage Admin__ _(Full control of GCS resources)_
-7. Click the __Create__ button. This will download a credentials file to your desktop. Let's call this credentials.json
+This topic describes how to use Minio GCS Gateway to add Amazon S3 compatibility to Google Cloud Storage (GCS).
```
Comment (poornas) [inline docs/gateway/gcs.md]: ```Minio GCS Gateway allows you to access Google Cloud Storage (GCS) with Amazon S3 compatible APIs``` seems better

**Review by poornas — CHANGES_REQUESTED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Review by eco-minio — APPROVED** [review]
LGTM

**Review by poornas — APPROVED** [review]

**Pull request merged**